src/main/scala/io/mesosphere/mesosaurus/Mesosaurus.scala

```diff
@@ -120,16 +121,16 @@ object Mesosaurus extends Logging {
         addOption(parser, MEM).`type`(UnsignedLong).help("mean # MB of memory")
         addOption(parser, MEM_SIGMA).`type`(UnsignedLong).help("memory standard deviation in MB")
         addOption(parser, PORT).`type`(UnsignedInteger).help("framework port to use")
-    }
+        addOption(parser, FAIL).`type`(UnsignedDouble).help("failure rate to use")
```

Comment:
Would prefer a more explicit name for this option.  Furthermore, what's the unit of measurement for this value?  e.g. mean-percent-failed-per-second, etc.